docs/PLUGIN_CATALOG.md

```diff
@@ -19,19 +19,20 @@ This file is automatically generated. If you would like to add to the plugin lis
 | [CEPH](https://github.com/intelsdi-x/snap-plugin-collector-ceph) | [Intel](https://github.com/intelsdi-x) | Collects OpenStack CEPH module metrics | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-ceph.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-ceph) | [release](https://github.com/intelsdi-x/snap-plugin-collector-ceph/releases) |
 | [Cinder](https://github.com/intelsdi-x/snap-plugin-collector-cinder) | [Intel](https://github.com/intelsdi-x) | Collects OpenStack Cinder module metrics | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-cinder.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-cinder) | [release](https://github.com/intelsdi-x/snap-plugin-collector-cinder/releases) |
 | [Couchbase](https://github.com/Staples-Inc/snap-plugin-collector-couchbase) | [Staples Inc](https://github.com/Staples-Inc) | Collects Bucket Stats from the CouchBase API using SNAP. |  | [release](https://github.com/Staples-Inc/snap-plugin-collector-couchbase/releases) |
-| [CPU](https://github.com/intelsdi-x/snap-plugin-collector-cpu) | [Intel](https://github.com/intelsdi-x) | Collects Linux cpu metrics from /proc/stat. | <ul><li>[![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-cpu.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-cpu)</li><li>[![Build Status](https://ci.snap-telemetry.io/buildStatus/icon?job=snap-plugin-collector-cpu-large-prb-k8s)](https://ci.snap-telemetry.io/job/snap-plugin-collector-cpu-large-prb-k8s/)</li></ul> | [release](https://github.com/intelsdi-x/snap-plugin-collector-cpu/releases) |
+| [CPU](https://github.com/intelsdi-x/snap-plugin-collector-cpu) | [Intel](https://github.com/intelsdi-x) | Collects Linux cpu metrics from /proc/stat. | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-cpu.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-cpu) | [release](https://github.com/intelsdi-x/snap-plugin-collector-cpu/releases) |
```

Comment:
@danieldn, according to https://github.com/intelsdi-x/snap/issues/1613 - is it possible to keep presenting the status of large tests?